src/plone/restapi/services/copymove/copymove.py

```diff
@@ -25,12 +23,8 @@ def __init__(self, context, request):
         self.catalog = getToolByName(self.context, 'portal_catalog')
 
     def get_object(self, key):
-        """Get an object by intid, url, path or UID."""
-        if isinstance(key, int):
```

Comment:
Removing the lookup by intid is fine with me. @csenger @sneridagh I think one of you guys added that, would you be ok with removing it, or do you have a strong opinion on keeping it? We can always add it properly at a later point if needed.